test/verify/check-machines

```diff
@@ -3730,7 +3733,32 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             ip_conf="IPv4 only",
             ipv4_address="192.168.110.0",
             ipv4_netmask="255.255.255.0",
+            remove=False,
+            activate=True,
+        ).execute()
+
+        # XFail: Activate a network which has a same ipv4 address with last one
+        NetworkCreateDialog(
+            self,
+            name="test_network_duplication",
+            forward_mode="nat",
+            ip_conf="IPv4 only",
+            ipv4_address="192.168.110.0",
+            ipv4_netmask="255.255.255.0",
+            activate=True
         ).execute()
+        # Verify the error message and clean up the network
+        b.wait_present("#app > div > section > div > div")
+        b.click("#app > div > section > div > div > h4 > button")
+        b.wait_in_text(
+            "#app > div > section > div > div > div.pf-c-alert__description > p",
+            "Network is already in use")
+        b.click(
+            "#app > div > section > div > div > div.pf-c-alert__action > button")
+        b.wait_not_present("#app > div > section > div > div")
+        m.execute(
```

Comment:
Please use the existing `cleanup` method.
You can adjust it to do `virsh destroy net-name` conditionally if `activate` was set.
I would also prefer to have the expected error checking inside a method.
You can name it `verify_expected_error` and just pass it the expected error string as argument.